src/fci/fci_vector.h

```diff
@@ -196,25 +196,14 @@ class FCIVector {
                 ncmo_ * ncmo_ * ncmo_ * r + ncmo_ * ncmo_ * s + ncmo_ * t + u);
     }
 
-    //    double oei_aa(size_t p, size_t q) const {return fci_ints_->oei_a(ncmo_
-    //    * p + q);}
-    //    double oei_bb(size_t p, size_t q) const {return fci_ints_->oei_b(ncmo_
-    //    * p + q);}
-
-    //    double tei_aaaa(size_t p, size_t q, size_t r, size_t s) const {return
-    //    fci_ints_->tei_aa(tei_index(p,q,r,s));}
-    //    double tei_aabb(size_t p, size_t q, size_t r, size_t s) const {return
-    //    fci_ints_->tei_ab(tei_index(p,q,r,s));}
-    //    double tei_bbbb(size_t p, size_t q, size_t r, size_t s) const {return
-    //    fci_ints_->tei_ab(tei_index(p,q,r,s));}
-
     void H0(FCIVector& result, std::shared_ptr<ActiveSpaceIntegrals> fci_ints);
     void H1(FCIVector& result, std::shared_ptr<ActiveSpaceIntegrals> fci_ints, bool alfa);
     void H2_aabb(FCIVector& result, std::shared_ptr<ActiveSpaceIntegrals> fci_ints);
     void H2_aaaa2(FCIVector& result, std::shared_ptr<ActiveSpaceIntegrals> fci_ints, bool alfa);
 
     void compute_1rdm(std::vector<double>& rdm, bool alfa);
     void compute_2rdm_aa(std::vector<double>& rdm, bool alfa);
+    // Compute the matrix elements of the rdm <a^+_{pa} a^+_{qb} a_{sb} a_ra> -> tei_index(p,q,r,s)
```

Comment:
```suggestion
    // Compute the matrix elements of the rdm <a^+_{pa} a^+_{qb} a_{sb} a_ra> -> 2rdm_index(p,q,r,s)
```

Reply:
I want to stick with `tei_index` here because that is the name of the function used in this class.